### PR TITLE
Using '%p' formatting for 'parent_window' strings

### PR DESCRIPTION
**Pull request by blumf**
Follow up to PR #80
Also, using correct '%lu' for 'object_handle' items.
Had to use barely documented wxSscanf function for translating back from string.

### CODE REVIEW

File: src/gui/GUIURIHandlerHelper.cpp [hunk 1]
```diff
@@ -35,9 +35,8 @@
 wxWindow* GUIURIHandlerHelper::getParentWindow(const URI& uri)
 {
     wxString ms = uri.getParam("parent_window");
-    wxWindow* res;
-    if (!wxSscanf(ms, "%p", &res)) {
-         return NULL;
-    }
-    return res;
+    unsigned long mo;
+    if (!ms.ToULong(&mo, 16))
```
Comment (mariuz) [inline src/gui/GUIURIHandlerHelper.cpp]: I used solution from @arvanus , could you test it

Comment (blumf) [reply src/gui/GUIURIHandlerHelper.cpp]: The problem with testing here is that it possibly will work currently, but it's not guaranteed to work universally.

There are two assumptions in this code: That an unsigned long is big enough to hold a pointer, and that the '%p' format produces an unadorned hex number string. Neither are guaranteed by the C++ standard.

On Win64 systems, unsigned long is 32-bit while pointers are 64:
https://en.cppreference.com/w/cpp/language/types
Yet, it's perfectly possible for test runs to only have pointer values <4GB by chance and pass. (As it happens, a quick test gave a >4GB pointer and does fail)

Basically, wxSscanf is the safe way to read a '%p' formatted strings.
